### ISSUE TITLE
Error "builder/amberfiles folder missing" in dev version during amber.build

### ISSUE THREAD

**Issue by GerardBCN** [Acellera/htmd#184]
```
bash-4.2$ more log.txt 
-I: Adding  /shared/gerard/HTMD/htmd/htmd/builder/amberfiles -I /shared/gerard/HTMD/htmd/htmd/builder/amberfiles/ -I /shared/gerard/HTMD/htmd/htmd/builder/amberfiles/ -I /share
d/gerard/HTMD/htmd/htmd/builder/amberfiles/  to search path.
 /shared/gerard/HTMD/htmd/htmd/builder/amberfiles -I /shared/gerard/HTMD/htmd/htmd/builder/amberfiles/ -I /shared/gerard/HTMD/htmd/htmd/builder/amberfiles/ -I /shared/gerard/HT
MD/htmd/htmd/builder/amberfiles/ : No such file or directory
```

**Comment by j3mdamas** [comment]
I've fixed this but I haven't pushed yet... Sorry... I'll do it right now.

**Comment by j3mdamas** [comment]
or maybe after tomorrow. I have to cherry pick is from other changes I am still working on.

**Comment by j3mdamas** [comment]
This is unrelated with what I thought. Where are you running the amber.build? Does that machine have access to /shared/gerard/HTMD/htmd/htmd/builder/amberfiles? Show me this tomorrow.

**Comment by tonigi** [comment]
I'll have a look with Gerard and the source source directory at hand

**Comment by j3mdamas** [comment]
It's fixed in 1926019c5e8d54584d7ec79d27d40c52ea3a18c9 and will be on next release (or in your case, just pull the new dev version)

**Comment by j3mdamas** [comment]
Thanks for @tonigi to pointing out what was wrong in my `call` of tleap.
